unpack-build-maven-plugin-itests/unpack-build-maven-plugin/pom.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>org.kie</groupId>
+    <artifactId>unpack-build-maven-plugin-itests</artifactId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>unpack-build-maven-plugin</artifactId>
```

Comment:
Maybe add here an explanation why this has the same GAV as the "real" unpack-build-maven-plugin. Another option would be to rename this to `-dummy` and build it before the `tests` module and set up a clean plugin execution to clean it afterwards. Then the `tests` module would run.